components/sst_importer/src/sst_importer.rs

```diff
@@ -1357,6 +1359,29 @@ impl<E: KvEngine> SstImporter<E> {
 
             let mut value = Cow::Borrowed(iter.value());
 
+            if rewrite_rule.ignore_after_timestamp != 0 {
+                let ts = Key::decode_ts_from(iter.key())?;
+                if ts > TimeStamp::new(rewrite_rule.ignore_after_timestamp) {
+                    iter.next()?;
+                    INPORTER_COMPACT_KEYS_COUNT
+                        .with_label_values(&["after"])
+                        .inc();
+                    continue;
+                }
+            }
+            if rewrite_rule.ignore_before_timestamp != 0 && cf_name == CF_WRITE {
```

Comment:
Prefer don't send `ignore_before_timestamp` in client side for default CF SSTs. That can be more flexible (say, records before `shifted_start_ts` in default CF can be safely deleted.)

Reply:
then I think maybe we should use two fields to represent the safe ignore ts.
because for default/write cf, the safe ignore ts is quite different.
WDYT?